### PR TITLE
migrating to Go modules

### PR DESCRIPTION
**Pull request by georgi-denchew**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,34 +1,21 @@
...
-
-# Set up our extra bits in the image
-RUN apk --no-cache --upgrade add \
-    git \
-    curl \
-    ca-certificates \
-    && update-ca-certificates
-
-RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+ENV BUILDINFO_PACKAGE="${ORG_PATH}/${PROJECT}/vendor/${ORG_PATH}/service-status-go/buildinfo."
```
Comment (karsov) [inline Dockerfile]: This is not the correct path when using modules.

File: .circleci/config.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     working_directory: /go/src/github.com/Financial-Times/upp-aggregate-healthcheck
     docker:
-      - image: circleci/golang:1.11
+      - image: circleci/golang:1
```
Comment (karsov) [inline .circleci/config.yml]: Should be "golang:1" only.

File: .circleci/config.yml [hunk 3]
```diff
@@ -15,22 +15,18 @@ jobs:
...
-          command: go build -v
+          command: go build -mod=readonly -v
       - run:
           name: Run Tests
           command: |
             mkdir -p ${CIRCLE_TEST_REPORTS}/golang
             mkdir -p ${CIRCLE_COVERAGE_REPORT}
-            go test -race -v | go-junit-report > $CIRCLE_TEST_REPORTS/golang/junit.xml
-            go test -race -cover -coverprofile=$CIRCLE_COVERAGE_REPORT/coverage.out
+            go test -mod=readonly -race -v | go-junit-report > $CIRCLE_TEST_REPORTS/golang/junit.xml
```
Comment (karsov) [inline .circleci/config.yml]: This can be one-liner now.

File: .circleci/config.yml [hunk 4]
```diff
@@ -15,22 +15,18 @@ jobs:
...
+            go test -mod=readonly -race -cover -coverprofile=$CIRCLE_COVERAGE_REPORT/coverage.out
```
Comment (karsov) [inline .circleci/config.yml]: Please update docker image to "docker:18" a few lines down.

File: .circleci/config.yml (hunk 2 above)
Comment (georgi-denchew) [reply .circleci/config.yml]: https://circleci.com/gh/Financial-Times/upp-aggregate-healthcheck/464

This happens when I use `golang:1`

Comment (karsov) [reply .circleci/config.yml]: The "Make go get work" step should be removed.

File: Dockerfile (hunk 1 above)
Comment (gkazakov111) [reply Dockerfile]: You should use this : 
`ENV BUILDINFO_PACKAGE="github.com/Financial-Times/service-status-go/buildinfo."`

**Comment by mitagg** [comment]
We will remove the Snyk integration as it is not working with go modules.

**Pull request merged**